.gitignore

```diff
@@ -54,3 +54,5 @@ tokens
 /spec/cypress/screenshots
 
 /config/credentials/staging.key
+
+/config/credentials/production.key
```

Comment:
This is fine, but you might consider changing to `/config/credentials/*.key` to make it even more foolproof

Reply:
like 10 lines higher we already have it 😂 
I just never thought about it cause creating credentials adds the one-by-one ignores :P

Reply:
Added by ECF-TECH-33